cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -262,7 +263,12 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             ortResultInput = ortResultInput.replacePackageCurations(curations)
         }
 
-        val packageConfigurationProvider = packageConfigurationOption.createProvider()
+        val repositoryPackageConfigurations = ortResultInput.repository.config.configurations.packageConfigurations
+        val optionPackageConfigurations = packageConfigurationOption.createProvider().getPackageConfigurations()
+
+        val packageConfigurationProvider = SimplePackageConfigurationProvider(
+            optionPackageConfigurations + repositoryPackageConfigurations
```

Comment:
Hint: The constructor can potentially throw an exception. Before it was possible to verify in PSV for the global configuration that an exception would not happen. Now it is not possible anymore, because the local configuration and global configuration may change independently.

I don't see a good solution for this, it's maybe just an aspect of this approach.